drivers/storage/portworx/util/util.go

```diff
@@ -303,6 +305,37 @@ func GetPortworxVersion(cluster *corev1alpha1.StorageCluster) *version.Version {
 	return pxVersion
 }
 
+// GetStorkVersion returns the stork version based on the image provided.
+// We first look at spec.Image. If that is not present or invalid semvar, then we fallback to an
```

Comment:
nit: `spec.Stork.Image`
Also update the comment as per the below review comment.